test/validators/some-member-of.spec.js

```diff
@@ -0,0 +1,101 @@
+import { expect } from 'chai';
+import validator from '../../lib';
+
+describe('Some MemberOf validator', () => {
+  const objectRules = {
+    pet: [
+      {
+        name: 'some-memberOf', params: [['dog', 'cat', 'fish']]
+      }
+    ]
+  };
+
+  it('should success for one item in the list', () => {
+    const rejectedInputs = [
```

Comment:
minor, this should be just `inputs` right? because it's not intended to be rejected by the validator